op_builder/comm.py

```diff
@@ -0,0 +1,66 @@
+import torch
+from .builder import CUDAOpBuilder
+
+MV2 = f'/opt/mvapich2/gdr/2.3.6/no-mpittool/no-openacc/cuda11.3/mofed5.4/mpirun/gnu9.3.0'
+OMPI = f'/usr/local/mpi'
+
+MPI_INCL = OMPI + '/include'
+MPI_LIB = '-L' + OMPI + '/lib -lmpi'
+
+CUDA = '/usr/local/cuda'
+CUDA_INCL = CUDA + '/include'
+CUDA_LIB = '-L' + CUDA + '/lib64 -lm -lcuda -lcudart'
```

Comment:
Where is the NCCL lib/include path set? Would it be possible to switch between different versions of NCCL? For example, when nvidia releases a new version of NCCL tomorrow can we more quickly add it in here than way for pytorch to be re-built with it 3 months later.